xscen/aggregate.py

```diff
@@ -174,9 +187,7 @@ def climatological_mean(
             else new_history
         )
         ds_rolling[vv].attrs["history"] = history
-
-    if to_level is not None:
-        ds_rolling.attrs["cat:processing_level"] = to_level
+    ds_rolling.attrs["cat:processing_level"] = to_level
```

Comment:
This doesn't fit the docstring, which says None is possible!
I'm guessing you want to change the docstring?

Reply:
Ah, true! I got confused because the default is a string, not `None`. I'll reverse that.